lib/publiccloud/basetest.pm

```diff
@@ -72,15 +73,10 @@ sub provider_factory {
     elsif ($args{provider} eq 'GCE') {
         $args{service} //= 'GCE';
         if ($args{service} eq 'GCR') {
-            $provider = publiccloud::gcr->new(
-                account => get_var('PUBLIC_CLOUD_GOOGLE_ACCOUNT'),
-                service_acount_name => get_var('PUBLIC_CLOUD_GOOGLE_SERVICE_ACCOUNT'),
-                project_id => get_var('PUBLIC_CLOUD_GOOGLE_PROJECT_ID'),
-                client_id => get_var('PUBLIC_CLOUD_GOOGLE_CLIENT_ID'),
-                region => get_var('PUBLIC_CLOUD_REGION', 'europe-west1-b'),
-                storage_name => get_var('PUBLIC_CLOUD_GOOGLE_STORAGE', 'openqa-storage'),
-                username => get_var('PUBLIC_CLOUD_USER', 'susetest')
```

Comment:
Why did you remove those parameters?

Reply:
We don't need to pass these parameters to have thihs working. See the creation of the GCE https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f78569bc3e849bcbd1eb0478af11f8a87e357525/lib/publiccloud/basetest.pm#L86